content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,250 @@
+to the product form, switch to the :guilabel:`Miscellaneous` tab, and the :guilabel:`Version` field
+will be incremented.
+
+.. _plm/eco/tablet-view:
+
+Create ECO from tablet view
+===========================
+
+:abbr:`ECOs (engineering change orders)` created in *tablet view* are intended for operators to
+directly suggest clearer assembly instructions. To create an :abbr:`ECO (engineering change order)`,
+navigate to the :guilabel:`Work Orders` tab on an ongoing manufacturing order. Click the
+:guilabel:`tablet icon` for the desired work order to open the tablet view of the operation.
+
+.. image:: engineering_change_orders/tablet-icon.png
+   :align: center
+   :alt: Find the tablet icon for each operation, second from the far right.
+
+Next, add an instructional step by clicking the :guilabel:`☰ (three horizontal lines)` icon in
+tablet view of an operation. Then, click the :guilabel:`Add a step` button.
+
+.. image:: engineering_change_orders/additional-options-menu.png
+   :align: center
+   :alt: Navigate to the "Add a Step" pop-up by clicking the three horizontal lines icon in tablet
+         view.
+
+In the :guilabel:`Title` field, enter a short step description. Next, in the
+:guilabel:`Instructions` text field, type the instructions of the step in greater detail.
+Optionally, add an image to the :guilabel:`Document` field. Once completed, finish by clicking the
+:guilabel:`Propose Change` button.
+
+.. example::
+   To propose an additional check for broken components, enter the details in the :guilabel:`Add a
+   Step` pop-up. Doing so creates an instructional quality control point that will be reviewed in
+   the following section.
+
+   .. image:: engineering_change_orders/add-a-step.png
+      :align: center
+      :alt: Fill out the *Add a Step* form to suggest an additional quality control point.
+
+View ECO
+--------
+
+To review the proposed changes, go to the `BOM Changes` ECO type from :menuselection:`PLM app -->
+Overview` and click on the card to view the contained :abbr:`ECOs (engineering change orders)`.
+Click on the newly created :abbr:`ECO (engineering change order)` in the `New` stage, and view a
+summary of the proposed changes in the :guilabel:`Operation Changes` tab. To implement the changes,
+click the :guilabel:`Revision` smart button.
+
+.. example::
+   The :abbr:`ECO (engineering change order)` adding another check for broken components is created
+   in the `BOM Changes` ECO type found in :menuselection:`PLM app --> Overview`. By default,
+   :abbr:`ECOs (engineering change orders)` created from tablet view are named with the
+   manufacturing order number  as a suggested form of reference.
+
+   .. image:: engineering_change_orders/view-BOM-change.png
+      :align: center
+      :alt: alt text
+
+In the new :abbr:`BoM (Bill of Material)`, switch to the :guilabel:`Operations` tab and select the
+:guilabel:`Show Instructions` icon (:guilabel:`☰ (three horizontal lines)`). Doing so opens a list
```

Comment:
```suggestion
:guilabel:`☰ (Show Instructions)` icon. Doing so opens a list
```
I like to use the symbol itself followed by the name of the symbol in parentheses (ex. settings).